tap_github/streams.py

```diff
@@ -196,6 +197,47 @@ class ReadmeStream(GitHubStream):
     ).to_dict()
 
 
+class ReadmeHtmlStream(GitHubStream):
```

Comment:
It might be useful to add a docstring to explain why there seems to be 2 streams for README :)

Reply:
done